client/my-sites/plan-features-2023-grid/components/features.tsx

```diff
@@ -4,6 +4,7 @@ import styled from '@emotion/styled';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 import { LoadingPlaceHolder } from '../../plans-features-main/components/loading-placeholder';
+import useIsCustomDomainAllowedOnFreePlan from '../../plans-features-main/hooks/use-is-custom-domain-allowed-on-free-plan';
```

Comment:
We shouldn't be importing this in here. Generate the value from plans-features-main and pass as **optional** prop to plans-features-2023-grid

Reply:
Extracted by https://github.com/Automattic/wp-calypso/pull/78825/commits/e20c4cd1be566658fea954b235ab43155e072884